src/main/resources/crafter/studio/upgrade/3.2.x/config/permission-mappings-config/permission-mappings-config-v14.xslt

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License version 3 as published by
+  ~ the Free Software Foundation.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
+
+    <!-- to keep the right formatting -->
+    <xsl:output method="xml" indent="yes" />
+    <xsl:strip-space elements="*"/>
+
+    <!-- copy all elements -->
+    <xsl:template match="node() | @*">
+        <!-- insert line breaks before comments -->
+        <xsl:if test="self::comment()">
+            <xsl:text>&#10;</xsl:text>
+        </xsl:if>
+        <xsl:copy>
+            <xsl:apply-templates select="node() | @*"/>
+        </xsl:copy>
+        <!-- insert line breaks after comments -->
+        <xsl:if test="self::comment()">
+            <xsl:text>&#10;</xsl:text>
+        </xsl:if>
+    </xsl:template>
+
+    <xsl:template match="permissions/role/rule/allowed-permissions">
```

Comment:
What about `site-admins`? They should also be able to edit the site name and site description.

Reply:
@jdrossl has this been addressed?

Reply:
yeah https://github.com/craftercms/studio/pull/2202/files?file-filters%5B%5D=.xslt#diff-5e8e31cb61906b9a672c96fa634071c8R53